src/bin/pgcopydb/cli_dump.c

```diff
@@ -107,7 +107,9 @@ cli_dump_schema_getopts(int argc, char **argv)
 		exit(EXIT_CODE_BAD_ARGS);
 	}
 
-	while ((c = getopt_long(argc, argv, "S:T:D:PrReFCNVvdzqh",
```

Comment:
We should have `N:` as snapshot has a required argument.

`{ "snapshot", required_argument, NULL, 'N' }`

I wonder if this means we never supported `pgcopydb dump schema --snapshot <snapshot_id>` properly.

Reply:
I see. Can we include a fix for the argument parsing in this PR then?

Reply:
Once we programmatically create the optstring values, it will fix this underlying issue as well.

This bug only exists in the line that I want to remove.